github/table_github_audit_log.go

```diff
@@ -0,0 +1,146 @@
+package github
+
+import (
+	"context"
+	"time"
+
+	"github.com/google/go-github/v45/github"
+	pb "github.com/turbot/steampipe-plugin-sdk/v3/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v3/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableGitHubAuditLog(ctx context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "github_audit_log",
+		Description: "Gets the audit log for an organization.",
+		List: &plugin.ListConfig{
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "organization", Require: plugin.Required},
+				{Name: "phrase", Require: plugin.Optional},
+				{Name: "include", Require: plugin.Optional},
+				{Name: "action", Require: plugin.Optional},
+				{Name: "actor", Require: plugin.Optional},
+				{Name: "created_at", Require: plugin.Optional, Operators: []string{">", ">=", "<", "<=", "="}},
+			},
+			Hydrate: tableGitHubAuditLogList,
+		},
+		Columns: []*plugin.Column{
+			{Name: "organization", Type: pb.ColumnType_STRING, Transform: transform.FromQual("organization")},
+			{Name: "phrase", Type: pb.ColumnType_STRING, Transform: transform.FromQual("phrase"), Description: "The search phrase for your audit events."},
+			{Name: "include", Type: pb.ColumnType_STRING, Transform: transform.FromQual("include"), Description: "The event types to include: web, git, all."},
+
+			// Top columns.
+			{Name: "id", Type: pb.ColumnType_STRING, Description: "The id of the audit event.", Transform: transform.FromField("DocumentID")},
+			{Name: "created_at", Type: pb.ColumnType_TIMESTAMP, Description: "The timestamp of the audit event.", Transform: transform.FromField("CreatedAt").Transform(convertTimestamp)},
```

Comment:
What's the difference between the timestamp in `CreatedAt` and the `Timestamp` property (https://github.com/google/go-github/blob/fd22ee93382e50d36592170f63c6a08d7c35ed8d/github/orgs_audit_log.go#L84)?